pythran/pythonic/include/numpy/unravel_index.hpp

```diff
@@ -0,0 +1,21 @@
+#ifndef PYTHONIC_INCLUDE_NUMPY_UNRAVEL_INDEX_HPP
+#define PYTHONIC_INCLUDE_NUMPY_UNRAVEL_INDEX_HPP
+
+#include "pythonic/include/types/ndarray.hpp"
+#include "pythonic/include/types/str.hpp"
+
+PYTHONIC_NS_BEGIN
+
+namespace numpy
+{
+  template <class E, class IntTy, size_t N>
+  typename std::enable_if<std::is_scalar<E>::value,
+                          types::array<IntTy, N>>::type
+  unravel_index(E const &expr, types::array<IntTy, N> const &shape,
```

Comment:
In order to match the new pshape scheme, ``types::array<IntTy, N>`` should be a ``Ss`` template, and N would be ``std::tuple_size<Ss>::value``